site/en/workflows.md

```diff
@@ -671,9 +794,16 @@ A line detection algorithm is run on every text region of every PAGE in the
 input file group, and a TextLine element with the resulting polygon
 outline is added to the annotation of the output PAGE.
 
-**Note:** If you use `ocrd-tesserocr-segment-line`, which uses only bounding boxes instead of polygon coordinates,
-then you should post-process with the processors described in [Step 12](#step-12-resegmentation-line-level).
-If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
+**Note:** If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
+
+**Note:** If you use `ocrd-tesserocr-segment-line`, which uses only bounding
+boxes instead of polygon coordinates, then you should post-process with the
+processors described in [Step 12](#step-12-resegmentation-line-level).
+_Alternatively_, consider using the all-in-one capabilities of
+[`ocrd-tesserocr-recognize`](#step-7-region-segmentation), which can do line segmentation
+and text recognition in one step by querying Tesseract's internal iterator
+(accessing the more precise polygon outlines instead of just coarse bounding
+boxes with lots of hard-to-recover overlap). _Alternatively_, run with `shrink_polygons=True` (accessing that same iterator to calculate convex hull polygons).
 
 **Note:** As described in [Step 7](#step-7-page-segmentation), `ocrd-sbb-textline-detector` and `ocrd-cis-ocropy-segment` do not only segment
 the page, but also the text lines within the detected text regions in one step. Therefore with those (and only with those!) processors you don’t
```

Comment:
```suggestion
the page, but also the text lines within the detected text regions in one step. Therefore with those (and only with those!) processors you can directly go on with Step 13.
```

Reply:
Leaving out step 11 itself (and step 12), because it does not apply anymore, is not the same as continuing directly with step 13. Also, your edit keeps the next line untouched (you can make multi-line suggestions by dragging the plus symbol downwards across a range of lines).

Reply:
ah, sry, misread it